plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/data/JDBCCollection.java

```diff
@@ -130,7 +130,7 @@ public String toString() {
     }
 
     @NotNull
-    public String makeArrayString() {
+    public String makeArrayString(char ... brackets) {
```

Comment:
Why is it a variadic-length argument? Pass two parameters: for the left bracket and the right one.